### PR TITLE
Transcription errors and editorial changes

### PR DESCRIPTION
**Pull request by jennfrye**
Just finished reading this and found a few errors that I’ve corrected. I was not able to view beyond Page 109 in [the source scans](https://archive.org/details/157028180.1846.emory.edu/page/78/mode/1up?q=helped) due to some sort of glitch on the part of Internet Archive, but I found another scan of [the same edition on Google Books](https://books.google.com/books?id=ey_lLa0zNDkC), so that is what I’ve referenced for non-editorial changes. I hope that’s okay.

`[Editorial] Add paragraph breaks in dialog` is a follow-up to an earlier editorial commit with the same commit message. The earlier commit divided what was originally a single paragraph with several lines of dialogue into multiple paragraphs (one paragraph per speaker), but missed dividing out the last line of dialogue: I corrected this and made a couple of other similar divisions that I noticed while reading for consistency.

### CODE REVIEW

File: src/epub/text/chapter-2-1-8.xhtml [hunk 1]
```diff
@@ -99,11 +99,11 @@
...
+				<p>There was something horrible⁠—something fierce and devilish⁠—in the outburst of his delight at his own singing and playing, and in the triumph with which he watched its effect upon me as I shrank nearer and nearer to the door. I was released at last, not by my own efforts, but by Sir Percival’s interposition. He opened the dining-room door, and called out angrily to know what “that infernal noise” meant. The Count instantly got up from the piano. “Ah! if Percival is coming,” he said, “harmony and melody are both at an end. The Muse of Music, Miss Halcombe, deserts us in dismay, and I, the fat old minstrel, exhale the rest of my enthusiasm in the open air!” He stalked out into the verandah, put his hands in his pockets, and resumed the “<span epub:type="se:name.music.song">recitativo of Moses</span>,” sotto voce, in the garden.</p>
```
Comment (acabal) [inline src/epub/text/chapter-2-1-8.xhtml]: `Recitativo` should be capitalized

**Review by acabal — CHANGES_REQUESTED** [review]
Looks great. Just one minor fix before I can merge this!

File: src/epub/text/chapter-2-1-8.xhtml (hunk 1 above)
Comment (jennfrye) [reply src/epub/text/chapter-2-1-8.xhtml]: Done! I made it as an editorial commit since it's not capitalized in the source scans.

**Comment by acabal** [comment]
Great, thanks!

**Pull request merged**